packages/react-devtools-shared/src/hook.js

```diff
@@ -302,7 +302,10 @@ export function installHook(target: any): DevToolsHook | null {
   const renderers = new Map();
 
   let hook: DevToolsHook;
-  if (!canAccessParentWindow(target) || isMainWindow(target)) {
+  if (canAccessParentWindow(target) && !isMainWindow(target)) {
+    hook = getMainWindow(target).__REACT_DEVTOOLS_GLOBAL_HOOK__;
+  }
+  if (!hook) {
```

Comment:
I handle know if we don't have `__REACT_DEVTOOLS_GLOBAL_HOOK__` in the parent window